dev/release/verify-release-candidate.sh

```diff
@@ -254,7 +254,13 @@ test_python() {
 test_glib() {
   pushd c_glib
 
-  ./configure --prefix=$ARROW_HOME
+  if [ "$(uname)" == "Darwin" ]; then
```

Comment:
How about `if brew --prefix libffi > /dev/null 2>&1; then`?
Some macOS environments may not install Homebrew.

Reply:
Done, thanks!